src/main/java/net/dv8tion/jda/internal/entities/AbstractChannelImpl.java

```diff
@@ -106,15 +106,12 @@ public JDA getJDA()
     }
 
     @Override
-    public PermissionOverride getPermissionOverride(Member member)
+    public PermissionOverride getPermissionOverride(IPermissionHolder permissionHolder)
     {
-        return member != null ? overrides.get(member.getUser().getIdLong()) : null;
-    }
+        if (permissionHolder == null)
+            return null;
```

Comment:
Do we want to maintain this functionality? Or should we throw `IllegalArgumentException` if the holder is null?

Reply:
I believe we _should_ throw.